### PR TITLE
Security Related Documentation Updated for IS-5.9.0

### PR DESCRIPTION
**Pull request by Buddhimah**

### CODE REVIEW

File: en/docs/administer/production-deployment-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,701 @@
+# Production Deployment Guidelines
```
Comment (ayshsandu) [inline en/docs/administer/production-deployment-guidelines.md]: I think we removed this intentionally, check with @somindatommy

File: en/docs/setup/installing-on-windows.md [hunk 2]
```diff
@@ -142,7 +142,6 @@ you can take the following approaches:
     code that reads the properties at runtime from that registry. Be
     sure to store sensitive data such as username and password to
     connect to the registry in a properties file instead of in the Java
-    code and secure the properties file with the [secure
-    vault](../../administer/carbon-secure-vault-implementation).
+    code and secure the properties file with the cipher tool](../../administer/encrypting-passwords-with-cipher-tool).
```
Comment (omindu) [inline en/docs/setup/installing-on-windows.md]:
```suggestion
    code and secure the properties file with the [cipher tool](../../administer/encrypting-passwords-with-cipher-tool).
```

File: en/docs/administer/production-deployment-guidelines.md (hunk 1 above)
Comment (somindatommy) [reply en/docs/administer/production-deployment-guidelines.md]: Yes, we removed this. Some content is this doc is already in other docs. And the content that are not in other docs was moved to new docs.

Comment (Buddhimah) [reply en/docs/administer/production-deployment-guidelines.md]: removed from doc space

File: en/docs/setup/installing-on-windows.md (hunk 2 above)
Comment (Buddhimah) [reply en/docs/setup/installing-on-windows.md]: done

**Pull request merged**